sdk/messaging/eventgrid/azsystemevents/constants.go

```diff
@@ -148,7 +172,6 @@ const (
 	ACSRouterUpdatedWorkerPropertyAvailableForOffers    ACSRouterUpdatedWorkerProperty = "AvailableForOffers"
 	ACSRouterUpdatedWorkerPropertyChannelConfigurations ACSRouterUpdatedWorkerProperty = "ChannelConfigurations"
 	ACSRouterUpdatedWorkerPropertyLabels                ACSRouterUpdatedWorkerProperty = "Labels"
-	ACSRouterUpdatedWorkerPropertyMaxConcurrentOffers   ACSRouterUpdatedWorkerProperty = "MaxConcurrentOffers"
```

Comment:
Should this, ChannelType -> ChannelKind, and the model field renames be in the changelog?

Reply:
Well, "should" was NOT true :) I've updated the changelog to incorporate all the renames. It wasn't too many. I also added in a rename for the RecordingChannelType (which was the only one I saw that hadn't gotten the rename treatment).